asv/environment.py

```diff
@@ -440,8 +440,11 @@ def get_environment_class_by_name(environment_type):
     for cls in util.iter_subclasses(Environment):
         if cls.tool_name == environment_type:
             return cls
+    tool_names = [cls.tool_name for cls in util.iter_subclasses(Environment)]
     raise EnvironmentUnavailable(
-        f"Unknown environment type '{environment_type}'")
+        f"Unknown environment type '{environment_type}'. "
+        f"Allowed values are {tool_names}."
+    )
```

Comment:
```suggestion
        f"Unknown environment type '{environment_type}'. "
        f"Allowed values on this system are {tool_names}."
    )
```

Change to highlight that not all the plugins will be visible, e.g. if `libmambapy` is not installed `mamba` will not show up as an allowed value. Feel free to rework this suggestion a bit (maybe to include a link to the documentation where plugins can be listed).